src/databricks/labs/ucx/assessment/queries/database_summary.sql

```diff
@@ -0,0 +1,26 @@
+-- viz type=table, name=Database Summary, columns=database,tables,views,dbfs_root,delta_tables,upgrade
+-- widget title=Database Summary, col=0, row=6, size_x=3, size_y=8
+SELECT `database`,
+       SUM(is_table) AS tables,
+       SUM(is_view) AS views,
+       SUM(is_dbfs_root) AS dbfs_root,
+       SUM(is_delta) AS delta_tables,
+       CASE
+           WHEN (SUM(is_dbfs_root)/SUM(is_table) > .3) THEN "Asset Replication Required"
+           WHEN (SUM(is_delta)/SUM(is_table) < .7) THEN "Some Non Delta Assets"
+           ELSE "In Place Sync"
+       END AS upgrade
+FROM
+  (SELECT DATABASE,
+          name,
+          object_type,
+          UPPER(table_format) AS format,
+          LOCATION,
+          IF(object_type IN ("MANAGED", "EXTERNAL"), 1, 0) AS is_table,
+          IF(object_type = "VIEW", 1, 0) AS is_view,
+          IF(STARTSWITH(location, "/dbfs/")
+             AND NOT STARTSWITH(location, "/dbfs/mnt"), 1, 0) AS is_dbfs_root,
```

Comment:
```suggestion
          IF(STARTSWITH(location, "/dbfs/") AND NOT STARTSWITH(location, "/dbfs/mnt"), 1, 0) AS is_dbfs_root,
```
to avoid confusion of reading it :) logic is expected

Reply:
I fixed #1 and #3, but not sure what #2 is about.